tests/test_psf_library.py

```diff
@@ -0,0 +1,207 @@
+import astropy.convolution
+import numpy as np
+import os
+import pytest
+import webbpsf
+from astropy.io import fits
+from mirage.psf.psf_library import CreatePSFLibrary
+
+
+# @pytest.mark.skip()
+def test_all():
```

Comment:
I might make this test name a little more descriptive, like `test_all_filters_and_detectors`